src/api/.haml-lint.yml

```diff
@@ -18,6 +18,8 @@ linters:
     enabled: true
     exclude:
       - 'app/views/webui/users/new.html.haml'
+      # Workaround bug: Couldn't process the file for linting. NoMethodError: undefined method `each_node' for nil:NilClass
```

Comment:
How are you tracking this "bug"?

Reply:
In the end it was not a bug. There was a syntax error somewhere else and the linter was choking on it